### PR TITLE
Code cleaning and fixes for jenkins

### PR DESCRIPTION
**Pull request by Michalk39**
I made a little code cleaning on this branch now.
Previously I made a lot of small fixes in the whole project, so if you can, please review the whole code.

### CODE REVIEW

File: pages/app/magentoCheckout.ts [hunk 1]
```diff
@@ -23,10 +23,7 @@ export class MagentoCheckout {
     }
 
     public async fillEmailAdressField(email: string) {
-        await CustomWait.waitForElementToBeClickable(
-            this.emailAdressField,
-            CustomWait.timeouts.long
-        );
+        await CustomWait.waitForElementToBeClickable(this.emailAdressField, CustomWait.timeouts.long);
```
Comment (MichalFidor) [inline pages/app/magentoCheckout.ts]: Won't work with the timeout which is provided in a method `waitForElementToBeClickable`?

File: pages/app/magentoContentPages.ts [hunk 2]
```diff
@@ -76,17 +72,13 @@ export class MagentoContentPages {
...
+        let checkBox = $("table[data-role='grid'] > tbody > tr:nth-child(" + number + ") > td:first-child");
         await Actions.click(checkBox);
     }
 
     public async clickRowCheckboxReversed(number: number) {
         await CustomWait.waitForElementToBeClickable(this.lastRowTitle);
-        let checkBox = $(
-            "table[data-role='grid'] > tbody > tr:nth-last-child(" + number + ") > td:first-child"
-        );
+        let checkBox = $("table[data-role='grid'] > tbody > tr:nth-last-child(" + number + ") > td:first-child");
```
Comment (MichalFidor) [inline pages/app/magentoContentPages.ts]: It's better to use concatenate with `text ${(some_variable)} text`. Please change it and remove `+`

Change it in all places where you use concatenate with `+` (not only in selectors). 
[Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Expression_interpolation) if you didn't use it earlier

File: pages/app/magentoContentPages.ts [hunk 3]
```diff
@@ -32,13 +32,9 @@ export class MagentoContentPages {
     }
 
     public async selectActionFromList(action: string) {
-        let listItem = await element(
-            by.cssContainingText("ul > li > span.action-menu-item", action)
-        );
+        let listItem = await element(by.cssContainingText("ul > li > span.action-menu-item", action));
```
Comment (MichalFidor) [inline pages/app/magentoContentPages.ts]: You added a lot of selectors to methods in this class. But you also have added earlier some of them into a constructor. Why you don't add them all into a constructor? 

Except these where you would like to use variables.

File: pages/app/magentoHomePage.ts [hunk 4]
```diff
@@ -42,16 +42,14 @@ export class MagentoHomePage {
...
-        // to by można było sparametryzować
         let productName: string = "Example Product";
         await this.navigateTo();
-        await browser.sleep(5000); // moze tutaj problem
+        await browser.waitForAngular();
         await this.searchPhrase(productName);
-        // await Actions.attachScreenshot(this);
-        await browser.sleep(5000); // moze tutaj problem
+        await CustomWait;
+        await browser.sleep(5000);
```
Comment (MichalFidor) [inline pages/app/magentoHomePage.ts]: Something isn't go well here ;)

File: pages/app/magentoHomePage.ts [hunk 5]
```diff
@@ -42,16 +42,14 @@ export class MagentoHomePage {
...
         await Actions.click(await element(by.cssContainingText(".product-item-link", productName)));
-        // await Actions.attachScreenshot(this);
-        await browser.sleep(5000); // moze tutaj problem
+        await browser.sleep(5000);
```
Comment (MichalFidor) [inline pages/app/magentoHomePage.ts]: And don't use `browser.sleep`. It's better to wait for some element or condition. Sleeping the browser in such an ordinary way is a finality.

File: pages/app/magentoCheckout.ts (hunk 1 above)
Comment (Michalk39) [reply pages/app/magentoCheckout.ts]: It happens that the page loads quite a long time. 
Definitely more than the default 3 seconds.

File: stepdefinitions/shopping-cart.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+const magentoHomePage: MagentoHomePage = new MagentoHomePage();
+const magentoCheckout: MagentoCheckout = new MagentoCheckout();
+const magentoProductsCatalog: MagentoProductsCatalog = new MagentoProductsCatalog();
+
+Given(/^User have non-empty shopping cart$/, async function() {
+    await Actions.attachScreenshot(this);
+    await magentoProductsCatalog.createExampeProductIfNotExist("Example Product");
+    await Actions.attachScreenshot(this);
+    await magentoHomePage.addProductToCart();
+    await Actions.attachScreenshot(this);
```
[the remark this answers is not in the document]
Comment (Michalk39) [reply stepdefinitions/shopping-cart.ts]: Yes, three images appear for that step in the report.

File: stepdefinitions/cms-pages.ts [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+});
+
+Then(/^Page should be visible in table$/, async function() {
+    await magentoContentPages.navigateTo();
+    expect(await magentoContentPages.getLastRowTitle()).equal(this.pageTitle);
+});
+
+Then(/^Page url should be reachable$/, async function() {
+    await BrowserActions.get(await magentoContentPages.getLastRowUrl());
+    expect(await $("li.item.cms_page:nth-child(2) > strong").getText()).equal(this.pageTitle);
```
[the remark this answers is not in the document]
Comment (Michalk39) [reply stepdefinitions/cms-pages.ts]: Totally agree, I don't know why i did this like that.

**Pull request merged**